tools/post-potd/post-potd.py

```diff
@@ -0,0 +1,37 @@
+from discord import SyncWebhook
+
+import json
+import requests
+
+from bs4 import BeautifulSoup
+
+
+def get_last_potd():
+    response = requests.get("https://leetcode.com/problemset/all/")
+    if response.status_code != 200:
+        raise "oops"
+    soup = BeautifulSoup(response.text, features="html.parser")
+    data = json.loads(soup.find(id="__NEXT_DATA__").text)
+    queries = data["props"]["pageProps"]["dehydratedState"]["queries"]
+    relevant_queries_data = [
```

Comment:
This formatting should be more readable.

Reply:
Fwiw I used [black](https://pypi.org/project/black/) for the formatting. But I guess these list comprehensions are a bit much.